tmva/tmva/src/MethodDL.cxx

```diff
@@ -1432,8 +1432,10 @@ void MethodDL::TrainDeepNet()
       Log() << "   Start epoch iteration ..." << Endl;
       bool debugFirstEpoch = false;
       bool computeLossInTraining = true;  // compute loss in training or at test time
+      size_t nTrainEpochs = 0;
```

Comment:
So the `nTrainEpochs` replaces now the global step for computing the validation, right?

Reply:
I added the function to test the resetting of time steps for every epoch, but it did not make sense to do it. 
I agree the function ResetGlobalSteps is not needed and can  be removed. 

Lorenzo